docs/tools/terraform/get-started.rst

```diff
@@ -88,64 +87,67 @@ The ``variables.tf`` file defines both the API token, and the project name to us
    }
    
    
-The ``var-values.tfvars`` file holds the actual values and is passed to Terraform using the ``-var-file=`` flag.
+6. Create a file named ``terraform.tfvars`` to define the values of the sensitive information. 
 
-``var-values.tfvars`` file:
+Add the following code, replacing ``AIVEN_AUTHENTICATION_TOKEN`` with your API token and ``AIVEN_PROJECT_NAME`` with the name of your project:
 
 .. code:: terraform
 
-   aiven_api_token = "<YOUR-AIVEN-AUTHENTICATION-TOKEN-GOES-HERE>"
-   project_name    = "<YOUR-AIVEN-CONSOLE-PROJECT-NAME-GOES-HERE>"
+   aiven_api_token = "AIVEN_AUTHENTICATION_TOKEN"
+   project_name    = "AIVEN_PROJECT_NAME"
    
-Edit the file and replace the ``<..>`` sections with the API token you created earlier, and the name of the Aiven project that resources should be created in.
-
 
-Apply the Terraform configuration
+Plan and apply the configuration
 '''''''''''''''''''''''''''''''''
 
-
-The ``init`` command performs several different initialization steps in order to prepare the current working directory for use with Terraform. In our case, this command automatically finds, downloads, and installs the necessary Aiven Terraform provider plugins.
+1. The ``init`` command prepares the working directly for use with Terraform. Run this command to automatically find, download, and install the necessary Aiven Provider plugins:
 
 .. code:: bash
 
    terraform init 
 
-The ``plan`` command creates an execution plan and shows you the resources that will be created (or modified) for you. This command does not actually create any resource; this is more like a preview.
+2. Run the ``plan`` command to create an execution plan and view the resources that will be created or modified. This command does not create any resources. 
 
 .. code:: bash
 
-   terraform plan -var-file=var-values.tfvars
+   terraform plan
 
-If you're satisfied with the output of ``terraform plan``, go ahead and run the ``terraform apply`` command which actually does the task or creating (or modifying) your infrastructure resources. 
+3. To create the resources, run:
 
 .. code:: bash
 
-   terraform apply -var-file=var-values.tfvars
+   terraform apply
+
+The output will be similar to the following:
+
+.. code:: bash
+  
+  Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
 
-The output will show you if everything worked well. You can now visit the `Aiven web console <https://console.aiven.io>`_ and admire your new services.
+You can also see the service in the `Aiven Console <https://console.aiven.io>`_.
```

Comment:
nit: Terraform will ask you to type "yes". Running `apply` without `--auto-approve` flag outputs confirmation:

```
Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes <- you type this "yes"
```